modules/ROOT/pages/migration/upgrading-ocis.adoc

```diff
@@ -23,7 +23,7 @@ IMPORTANT: When upgrading from an older release to the desired one, *ALL* upgrad
 
 === Breaking Changes Requiring Manual Intervention
 
-* All environment variables that were marked for deprecation in Infinite Scale release 5.0.0 have finally been removed.
+* All environment variables that were marked for deprecation in Infinite Scale release 4.0.0 have finally been removed.
```

Comment:
hmm. wording is hard. We marked them to be removed in 5.0.0.

Reply:
Good point. I just prepare a commit with other stuff and fix it there.

Reply:
`All environment variables that were marked for deprecation in Infinite Scale release 5.0.0` <-- this part of the sentence is wrong or at least very easy to misunderstand.

Reply:
But the sentence says:
`All environment variables that were marked for deprecation in Infinite Scale release 4.0.0` ??

Reply:
Ah, the diff looked like it was only a suggestion and would currently be 5.0.0. Nevermind then.

Reply:
setting to resolve 😄